data/remnant/remnant missions.txt

```diff
@@ -3951,35 +3951,30 @@ event "remnant: postverta reveal"
 		link Diespiter
 		link Prosa
 		link Statina
-	system Diespiter
-		link Postverta
-		link Statina
-	system Statina
-		link Postverta
-		link Diespiter
-	system Levana
-		add object wormhole_barren
-			distance 2313
-			period 389
-			sprite planet/wormhole-red
 	system Caeculus
-		add object wormhole_barren2
+		add object "wormhole barren beta"
+			sprite planet/wormhole-red
 			distance 2400
 			period 391
+	system Egeria
+		add object "wormhole barren alpha"
 			sprite planet/wormhole-red
-	system Prosa
-		link Postverta
-		link Vaticanus
-		add object wormhole_barren2
 			distance 2313
 			period 389
+	system Levana
+		add object "wormhole barren alpha"
 			sprite planet/wormhole-red
-	system Egeria
-		add object wormhole_barren
 			distance 2313
 			period 389
+	system Prosa
+		add object "wormhole barren beta"
```

Comment:
You'll need to rename the planet definitions in map.txt as well. And object names should be capitalized; "Wormhole Barren Beta," not "wormhole barren beta."

Reply:
Thanks for the reminder. Done.